python/cudf/cudf/tests/test_list.py

```diff
@@ -460,15 +471,21 @@ def test_contains_invalid(data, scalar):
         ),
     ],
 )
-def test_index(data, scalar, expect):
+def test_index(data, search_key, expect):
```

Comment:
Can we add a test case for multi-level nested data? (if that is supported)

```python
sr = cudf.Series({"a": [[[1, 2], [3, 4]], [[5, 6], [7, 8]]]})
sr.list.index([[1, 2], [7, 8]])  # returns [0, 1]
```

Reply:
Still curious about multi-level nesting here. If multi-level nesting is supported, we'll need to revise a few other items as well. e.g. `is_scalar` might not be the appropriate check if "list scalars" are provided to check against a list of lists -- scalar-like input would have one fewer dimension / nested level that the input column, while column-like input would have an equal number of nested levels.